vendor/corssystem/cortu-gen-3.yaml

```diff
@@ -0,0 +1,192 @@
+name: Wind Sensor # Device name can not contain the vendor name
```

Comment:
It seems the majority of files you added contain the example wind sensor files. Could you fix this?

Reply:
Hi, I've fixed it, the file was still being drafted